src/elements/element.rectangle.js

```diff
@@ -69,7 +163,7 @@ module.exports = Element.extend({
 			bottom = vm.base;
 			signX = 1;
 			signY = bottom > top ? 1 : -1;
-			borderSkipped = vm.borderSkipped || 'bottom';
+			borderSkipped = valueOrDefault(vm.borderSkipped, 'bottom') || '';
```

Comment:
Do you need the `|| ''`? It seems like it'd never get used because it'd fall back to `'bottom'`

Reply:
No, removed